tensorflow/python/keras/layers/convolutional.py

```diff
@@ -77,6 +77,16 @@ class Conv(Layer):
       the dilation rate to use for dilated convolution.
       Currently, specifying any `dilation_rate` value != 1 is
       incompatible with specifying any `strides` value != 1.
+    groups: Integer, the number of channel groups controlling the connections
+      between inputs and outputs. Input channels and `filters` must both be
+      divisible by `groups`. For example,
+        - At `groups=1`, all inputs are convolved to all outputs.
+        - At `groups=2`, the operation becomes equivalent to having two
+          convolutional layers side by side, each seeing half the input
+          channels, and producing half the output channels, and both
+          subsequently concatenated.
+        - At `groups=input_channels`, each input channel is convolved with its
+          own set of filters, of size `input_channels / filters`
```

Comment:
should this say `filters / input_channels`?

Reply:
Good catch :+1: Fixed in c87915e09e5d9db8eae7b017307fc4b22862acc9